src/GUI/EFCorePowerTools/Extensions/ProjectExtensions.cs

```diff
@@ -79,15 +79,24 @@ public static List<string> GetConfigFiles(this Project project)
             return result.OrderBy(s => s).ToList();
         }
 
-        public static string GetRenamingPath(this Project project)
+        public static string GetRenamingPath(this Project project, string optionsPath)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            var projectPath = project.Properties.Item("FullPath")?.Value.ToString();
+            string projectPath;
```

Comment:
Rename to "renamingPath" ?

Reply:
done @ErikEJ  !

I originally kept it in order to respect previous code :)